_posts/en/newsletters/2025-10-17-newsletter.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Bitcoin Optech Newsletter #376'
+permalink: /en/newsletters/2025/10/17/
+name: 2025-10-17-newsletter
+slug: 2025-10-17-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+FIXME:bitschmidty
+
+## News
+
+- **Continued discussion of block template sharing:** Discussion continued after Anthony Towns [posted][towns tempshare] to Delving Bitcoin to suggest
+  full node peers occasionally send each other their current template
+  for the next block using [compact block relay][topic compact block
+  relay] encoding (see [Newsletter #366][news366 block template sharing]).
+  Since the last time we covered the issue, Towns opened [PR1937][pr 1937 bip] in the bips repository to formally propose [BIP153][stbip153]. However, due to privacy and node fingerprinting concerns raised by other developers, the author decided to move the current draft to the [Bitcoin Inquisition Numbers and Names Authority][binana repo] (BINANA) repository, to address these considerations and to refine the document. The draft was given the code [BIN-2025-0002][bin].
+
+- **B-SSL a Secure Bitcoin Signing Layer:** Francesco Madonna [posted][francesco post] to delving
+  about a concept which is a covenantless vault model using [taproot][topic taproot],
+  [CSV][op_csv], and [CLTV][op_cltv]. In the post, he mentions that it uses existing Bitcoin
+  primitives, which is important because most vault proposals require a soft fork.
+
+  In the design, there are three different spend paths.
+
+  1. A fast path for normal operation where an optional CS (Convenience Service)
+  can enforce the chosen delay.
+
+  2. A one-year fallback with custodian B
+
+  3. A three-year custodian path in case of disappearance or inheritance events.
+
+  There are 6 different keys A, A₁, B, B₁, C and CS where B₁, and C are
+  custodially held and are only used at the same time the recovery path.
+
+  This setup creates an environment where the user can lockup
+  their funds and be fairly sure that the custodians they have
+  entrusted their funds to won’t steal. While this does not restrict
+  where the funds can move to how a covenant would, this setup does
```

Comment:
```suggestion
  where the funds can move to like a covenant would, this setup does
```

Reply:
resolved in [ba32c10](https://github.com/bitcoinops/bitcoinops.github.io/pull/2527/commits/ba32c10a20d815329e23db570675264d1219ee0e)